dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_22__Add_ApiToken_Table.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE IF NOT EXISTS api_token
     lastUpdatedBy bigint                      not null,
     createdBy     bigint                      not null,
     version       int4                        not null,
-    type          int4                        not null,
+    type          character varying(50)       NOT NULL,
```

Comment:
I guess this will lead to issues with existing development databases? Not a problem, we should just give a heads up.

Reply:
Yes, this needs a DB "reset". I will announce it on #team-backend when it gets merged.

Reply:
You forgot @netroms :joy: